src/lib/features/like-episode/model/like.ts

```diff
@@ -0,0 +1,51 @@
+import type { Episode } from '$lib/shared/api';
+import { persistentWritable, localStorageAdapter } from 'svelte-persistent-writable';
+
+/**
+ * The store containing set of episodes liked by the current user.
+ */
+export const likesStore = persistentWritable(new Map<Episode['id'], Episode>(), {
```

Comment:
[question](https://conventionalcomments.org/#labels): are you sure we need to store the episodes here? I imagine we're going to have a separate store for episodes

Reply:
If you go through the commit history (in particular, https://github.com/aabounegm/cast/pull/66/commits/996f810507f43fa8e18cd5305316fcc42db84d76) you can see that I wanted to store only the episode ID at first. Then, when working on the "My library" page, I realized that we still don't have a store for the episodes with which I can use the IDs, so I changed it to store them as well to be able to display them directly in the favorites section of the library.

Reply:
But do you intend to change it later to only store the IDs, when we get our episode store in order?

Reply:
Hopefully yes, or at least the service that fetches them from the server